### PR TITLE
Install steps for default declaration directory, man page, and binary

### PR DESCRIPTION
**Pull request by jrroman**
…well as the man page, and the karn binary

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -60,10 +66,26 @@ vet: ## Verifies `go vet` passes
...
 
+.PHONY: create-declarations-dir
+create-declarations-dir:
+	@echo "Creating declarations dir at ~/.karn/declarations"
+	@mkdir -p ${INSTALLDECLARATIONS}
+	@cp -r ${DECLARATIONDIR} ${INSTALLDECLARATIONS}
+
+.PHONY: install-man
+install-man:
+	@echo "installing man page"
```
Comment (GrantSeltzer) [inline Makefile]: Can you capitalize the 'i' for consistency?

Comment (jrroman) [reply Makefile]: of course un momento porfa

Comment (jrroman) [reply Makefile]: the 'i' in the echo statement i assume?

**Comment by jrroman** [comment]
@GrantSeltzer here ya go, and not a problem my man, ill keep chippin away at some of these issues.

**Comment by GrantSeltzer** [comment]
Dope! LGTM!

**Comment by GrantSeltzer** [comment]
Dope! LGTM!

**Pull request merged**